setup.py

```diff
@@ -56,6 +56,6 @@
         "aiohttp>=3.8.3",
         "requests>=2.7.0",
         "pydantic>=1.10.2",
-        "pyjwt[crypto]>=2.3.0",
+        "pyjwt[crypto]>=2.4.0",
```

Comment:
Oh, that reminds me: we'll probably be able to close #74 and #83 after this merges

Reply:
Yeah, I should've mentioned, I looked into the optional features and couldn't figure out an easy way to divorce the `aiohttp` library completely. For now, I think this is the best I can do. There's probably some way to conditionally exclude the `aiohttp` library, but I don't know what patterns there are typical.